cmd/restore.go

```diff
@@ -54,9 +56,10 @@ type restoreCommand struct {
 	// to other controller nodes.
 	manualAgentControl bool
 
-	ui        *UserInteractions
-	restorer  *core.Restorer
-	converter func(member core.ReplicaSetMember) core.ControllerNode
+	ui              *UserInteractions
+	restorer        *core.Restorer
+	converter       func(member core.ReplicaSetMember) core.ControllerNode
+	readOneCharFunc func(*cmd.Context) (string, error)
```

Comment:
Maybe just readOneChar?

Reply:
Force of habit... In juju/juju there is a great insistence on suffixing func variables with an "F" or a "Func" :D